src/themes/creator.js

```diff
@@ -256,27 +256,33 @@ export default function createTheme(
       },
       font700: {
         fontFamily: primitives.primaryFontFamily,
-        fontSize: '28px',
+        fontSize: '32px',
         fontWeight: 'bold',
-        lineHeight: '40px',
+        lineHeight: '48px',
       },
       font800: {
         fontFamily: primitives.primaryFontFamily,
-        fontSize: '32px',
```

Comment:
We have a separate primitives list that are used for uber move theme creation.

Reply:
Yup! For type, the move and non-move primitives should be almost identical. Design isn't going to support this separate type ramp moving forward, and ideally the OS and non-OS versions should look as close to each other as possible.